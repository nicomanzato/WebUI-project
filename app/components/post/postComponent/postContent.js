import React, { Component } from 'react';
import {
  StyleSheet,
  View,
  Text,
  Image,
  TouchableHighlight
} from 'react-native';

export const PostContent = (props) => {
  return (
    <Text style={styles.postContent}>
      {props.post.text}
    </Text>
  );
}

const styles = StyleSheet.create({

  postContent:{
    fontSize: 15,
    fontWeight: "400"
  },

});
