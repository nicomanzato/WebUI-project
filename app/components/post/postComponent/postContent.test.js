import React from 'react'
import {PostContent} from './postContent'
import PostMock from './../../../store/post/mock/postMock'
import { shallow, mount, ReactWrapper } from 'enzyme'

describe('post component', () => {

  it('should render correctly', () => {
    const wrapper = shallow(
      <PostContent post={PostMock} />
    );

    expect(wrapper).toMatchSnapshot();
  });

});
