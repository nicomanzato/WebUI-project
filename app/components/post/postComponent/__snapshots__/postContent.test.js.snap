// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`post component should render correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "fontSize": 15,
      "fontWeight": "400",
    }
  }
>
  Por una medida de Trump, Rusia advirtió que "hay peligro" de una guerra nuclear: "Puede morir toda la civilización"… https://t.co/WG9oBkfoFF
</Text>
`;
